banking.c

```diff
@@ -0,0 +1,51 @@
+#include <stdio.h>
+
+typedef struct {
+	int acct_num;
+	char name[100];
+	float balance;
+} bank_acct;
+
+void print_details (bank_acct acct)
+{
+	printf("BANK ACCOUNT %d OWNED BY %s HAS $%.2f.\n", acct.acct_num, acct.name, acct.balance);
+}
+
+float withdraw (bank_acct acct)
+{
+	printf("Withdraw how much from account 2? ");
```

Comment:
You should be printing the account number from the structure instead of hardcoding it (same for your deposit function).

Also your name should be in a comment at the top of your code.  Otherwise, your code is very clean and straightforward - well done.